examples/contentful-simple/README.md

```diff
@@ -0,0 +1,247 @@
+# Makeswift + Contentful Integration
+
+This integration demonstrates how to combine the power of Contentful's content management with Makeswift's visual page building capabilities. Build visually editable blog websites where content is managed in Contentful but layouts are designed in Makeswift.
+
+## Prerequisites
+
+- [Node.js](https://nodejs.org/) 20.x or later
+- [Makeswift](https://www.makeswift.com/) account (for visual page building) [Sign up for free](https://app.makeswift.com/signup)
+- [Contentful](https://www.contentful.com/) account (for content management) [Sign up for free](https://www.contentful.com/sign-up)
+- Basic familiarity with [Next.js App Router](https://nextjs.org/docs/app)
+
+This guide assumes that you're familiar with Makeswift and setting up a custom host. Read here to learn more: https://docs.makeswift.com/developer/app-router/installation
+
```

Comment:
Since we're using VIBES pretty extensively here, I'd consider adding a short note/intro on it, something along the lines of

This integration also makes heavy use of UI components from the [Soul VIBE](https://vibes.site/docs/soul/introduction). Check out [vibes/README.md](vibes/README.md) for more details.